catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/components/hyperledger-aries.json

```diff
@@ -0,0 +1,37 @@
+{
+  "component": {
+    "kind": "component",
+    "name": "hyperledger-aries",
+    "title": "Aries",
+    "description": "Access market data and trade on Bitcoin and Altcoin exchanges.",
+    "deprecated": false,
+    "firstVersion": "3.17.0",
```

Comment:
First version should be 3.19.0 I suppose, because 3.18.0 is on vote already

Reply:
ok